addons/point_of_sale/static/src/app/store/models.js

```diff
@@ -1632,6 +1632,13 @@ export class Order extends PosModel {
             ),
         };
     }
+    _getProductNameWithVariantDetails(orderLine) {
```

Comment:
to me this method would have more sense to be on the orderline model.
Then you would be able to have 
```js
line.getFullName()
```